governance/program/src/lib.rs

```diff
@@ -6,8 +6,12 @@ pub mod error;
 pub mod instruction;
 pub mod processor;
 pub mod state;
+pub mod tools;
 
 // Export current sdk types for downstream users building with a different sdk version
 pub use solana_program;
 
 solana_program::declare_id!("Governance111111111111111111111111111111111");
```

Comment:
Might be time to start grinding for a real program id to use here

Reply:
Done